flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/catalog/CatalogTableITCase.scala

```diff
@@ -985,6 +985,80 @@ class CatalogTableITCase(isStreamingMode: Boolean) extends AbstractTestBase {
     assertEquals(false, tableSchema2.getPrimaryKey.isPresent)
   }
 
+  @Test
```

Comment:
If the table is created by TableAPi. The show create table result is strange, e.g.

```
Table t =
                tEnv.fromDataStream(
                        env.fromCollection(
                                Arrays.asList(
                                        new Tuple2<>(1, "1"),
                                        new Tuple2<>(1, "1"),
                                        new Tuple2<>(2, "3"),
                                        new Tuple2<>(2, "5"),
                                        new Tuple2<>(3, "5"),
                                        new Tuple2<>(3, "8"))),
                        $("id1"),
                        $("id2"),
                        $("proctime").proctime());

        tEnv.createTemporaryView("T", t);
```

```
CREATE TEMPORARY VIEW `default_catalog`.`default_database`.`T` AS
DataStream: (id: [default_catalog.default_database.T], fields: [id1, id2, proctime])
```

Reply:
That's a good point. I went through `TableEnvironmentImpl#createTemporaryView` and found that view is transformed to `QueryOperationCatalogView`, and in its constructor, the `QueryOperation#asSummaryString` is passed to `originalQuery` and `expandedQuery`, which causes this problem. And meanwhile, not all tables created by Table API encountered this problem, take `org.apache.flink.table.planner.utils.TestTableSourceSinks#createPersonCsvTemporaryTable` for example

```sacla
def createPersonCsvTemporaryTable(tEnv: TableEnvironment, tableName: String): Unit = {
    tEnv.connect(new FileSystem().path(getPersonCsvPath))
      .withFormat(
        new OldCsv()
          .fieldDelimiter("#")
          .lineDelimiter("$")
          .ignoreFirstLine()
          .commentPrefix("%"))
      .withSchema(
        new Schema()
          .field("first", DataTypes.STRING)
          .field("id", DataTypes.INT)
          .field("score", DataTypes.DOUBLE)
          .field("last", DataTypes.STRING))
      .createTemporaryTable(tableName)
  }
```
will generate
```sql
CREATE TEMPORARY TABLE `default_catalog`.`default_database`.`MyTable` (
  `first` STRING,
  `id` INT,
  `score` DOUBLE,
  `last` STRING
) WITH (
  'connector.path' = '/var/folders/xd/9dp1y4vd3h56kjkvdk426l500000gn/T/csv-test3761923201962233845tmp',
  'connector.property-version' = '1',
  'format.type' = 'csv',
  'format.line-delimiter' = '$',
  'format.ignore-first-line' = 'true',
  'format.property-version' = '1',
  'connector.type' = 'filesystem',
  'format.field-delimiter' = '#',
  'format.comment-prefix' = '%'
)
```
The reason is that the table created via `ConnectTableDescriptor` is a type of `CatalogTableImpl`, which is the same as the one that is created via SQL(please correct me if I'm wrong). 

Shall we revisit the decision that throwing an exception for the table which is created by Table API?

Reply:
Hi Shengkai, as discussed with Jark, I think we do not need to support `SHOW CREATE TABLE statement for view in this PR.

Reply:
@LadyForest @wuchong I have implemented  `SHOW CREATE TABLE` from hive catalog, but remains an issue to be solved:
I can't get the original order when I generate "WITH" information. Is there a way to solve it?